google/cloud/spanner/database_admin_client.h

```diff
@@ -126,7 +126,7 @@ class DatabaseAdminClient {
    *   completes on the service. Note that this can take minutes in some cases.
    *
    * @par Example
-   * @snippet database_admin_samples.cc create-database
+   * @snippet samples.cc create-database
```

Comment:
Why the rename?  We are going to have dozens, maybe as many as 100 samples. The files get unwieldy if they have too many of them.  Why not simply create a new sample, say `data_access_samples.cc` or something?

Reply:
I'll explain our rationale, then I'd love to hear if you have an alternate suggestion.

We discussed this locally yesterday. The thinking was that all of our samples will need some of the same infrastructure, like the ability to create a random database name, create the database, delete the database, some machinery to provide a command-line interface for executing the samples, etc. We could split that into a `database_admin_samples.cc` file and some *non-admin-samples.cc* file, or into the admin samples and N other files for different types of samples. Of course, we could also factory out common parts that would be shared by all the different samples flies, too.

We weren't sure about the best way to split up the samples. So we decided to put all the samples in a single file. That's the easiest solution for now, requiring little more than renaming the file to something not admin specific. Then if/when the `samples.cc` file becomes too big or unwieldy we'll be in a better position to split it up logically.

So, that's what we were thinking. Given that you have more experience with this, I'd really like to hear your thoughts.

Reply:
Ack. Splitting later seems fine.